namerd/iface/control-http/src/test/scala/io/buoyant/namerd/iface/HttpControlServiceTest.scala

```diff
@@ -285,6 +285,27 @@ class HttpControlServiceTest extends FunSuite with Awaits {
     assert(resp.contentString == bound + "\n")
   }
 
+  test("bind with an extra dtab") {
+    val ni = new NameInterpreter {
+      override def bind(dtab: Dtab, path: Path): Activity[NameTree[Bound]] = {
+        assert(dtab.show == "/foo=>/bar")
+        val (act, witness) = Activity[NameTree[Bound]]()
+        witness.notify(Return(NameTree.Leaf(Name.Bound(Var(null), s"/io.l5d.namer${dtab.lookup(path).show}"))))
+        act
```

Comment:
Could use `Activity.value(...)` here instead of creating an activity with a witness.